src/test/resources/marc_field999_empty.jsonl

```diff
@@ -0,0 +1,1118 @@
+{
```

Comment:
suggest to make all test cases more compact
to other words contain minimum subfield inside

Reply:
Fixed.